spec/src/main/asciidoc/SOAPAttachments.adoc

```diff
@@ -455,11 +452,11 @@ _javax.activation.DataHandler_ class and the Java™ Activation Framework.
 
  _multipart/*_
 
- _javax.mail.internet.MimeMultipart_
+ _jakarta.mail.internet.MimeMultipart_
 
  _text/xml_ or _application/xml_
 
- _javax.xml.transform.Source_
+ _jakarta.xml.transform.Source_
```

Comment:
this has not been moved to Eclipse and it remains in JDK, so this should stay as _javax.xml.transform.Source_